versions.tf

```
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = "~> 0.12"
+  required_providers {
+    google = "~> 2.7.0"
```

These restrictions should be loosened so users can leverage newer versions.

```suggestion
    google = "~> 2.7"
```

---

@aaron-lane, thank you for clarification! [Fixed](https://github.com/terraform-google-modules/terraform-google-lb-http/compare/42602b96de3e1eaa0f974e3f613a228edf64d322..c9213ad6284c1e30e30bd142519f80ab3f81e76c)